src/Test.UnitTests.WorkItems/FilingClientFactoryTests.cs

```diff
@@ -5,10 +5,10 @@
 using FluentAssertions;
 using Xunit;
 
-namespace Microsoft.WorkItemFiling
+namespace Microsoft.WorkItems
 {
-    public class FilingTargetFactoryTests
-    {
+    public class FilingClientFactoryTests
+    {       
```

Comment:
>        [](start = 5, length = 7)

Extra end-of-line white space was added.

Reply:
We need editorconfig! :)

---
In reply to: [381018726](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381018726) [](ancestors = 381018726)

Reply:
[https://github.com/microsoft/sarif-sdk/issues/1776](https://github.com/microsoft/sarif-sdk/issues/1776)

---
In reply to: [381450833](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381450833) [](ancestors = 381450833,381018726)